benchmarks/src/main/java/org/questdb/LineTCPSenderMainDupCols.java

```diff
@@ -37,7 +37,7 @@ public static void main(String[] args) {
 
         final Rnd rnd = new Rnd();
         long start = System.nanoTime();
-        try (LineTcpSender sender = LineTcpSender.newSender(Net.parseIPv4(hostIPv4), port, bufferCapacity)) {
+        try (AbstractLineTcpSender sender = AbstractLineTcpSender.newSender(Net.parseIPv4(hostIPv4), port, bufferCapacity)) {
```

Comment:
how are creating instances of abstract class? If it is not abstract, the class name must not contain "Abstract"